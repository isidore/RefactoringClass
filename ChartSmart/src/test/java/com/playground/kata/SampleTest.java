package com.playground.kata;

import org.approvaltests.Approvals;
import org.approvaltests.legacycode.Range;
import org.junit.Test;

public class SampleTest {

	@Test
	public void test() {
		Approvals.verifyAll("Numbers", Range.get(4, 8));
	}

}
