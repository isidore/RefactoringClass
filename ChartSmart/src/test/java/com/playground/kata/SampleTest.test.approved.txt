Numbers[0] = 4
Numbers[1] = 5
Numbers[2] = 6
Numbers[3] = 7
Numbers[4] = 8
